_posts/2019-07-12-mtls-with-nginx.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+current: post
+cover: datacenter.png
+title: Configuring Nginx with client certificate authentication (mTLS)
+date: 2019-07-12 12:00:00
+category: tutorials
+author: Viktor Petersson
+tags: [Open Source]
+class: post-template
+---
+
+#  Configuring Nginx with client certificate authentication (mTLS)
+
+**Required Skill Level**: Medium to Expert
+
+One of the cornerstones of Zero Trust Networking is Mutual TLS (known as mTLS). In simple terms, this means that each client is required to present a certificate to talk to the server. This is different compared to how your client (e.g. your web browser) only verifies the identity of the server. The client itself does not need to present any identity. The identity piece is normally solved using some kind of credential (such as a username/password or API token).
```

Comment:
Maybe just 'this is different compared to how your web browser works which only verifies the identity of the server.'

Reply:
I was trying to make it non-browser specific, to include use cases like an API for instance.